modules/Bio/EnsEMBL/Analysis/Hive/Config/Projection_subpipeline_conf.pm

```diff
@@ -0,0 +1,518 @@
+      -parameters => {
+        cmd => 'perl '.$self->o('flag_potential_pseudogenes_script').
+        ' -host '.$self->o('projection_db','-host').
+        ' -port '.$self->o('projection_db','-port').
+        ' -user_w '.$self->o('projection_db','-user').
+        ' -pass '.$self->o('projection_db','-pass').
+        ' -dbname '.$self->o('projection_db','-dbname').
+        ' -dna_host '.$self->o('dna_db','-host').
+        ' -dna_port '.$self->o('dna_db','-port').
+        ' -user_r '.$self->o('dna_db','-user').
+        ' -dna_dbname '.$self->o('dna_db','-dbname'),
+      },
+      -rc_name => '2GB',
+      -flow_into  => {
+        1 => ['fix_projection_db_issues'],
+      },
+    },
+
+    {
+      # This will fix issues when proteins that were too long for MUSCLE alignment didn't have proper ENST accessions
+      # Will also update the transcript table to match the gene table once the pseudo/canon genes are tagged in the
+      # previous analysis
+      -logic_name => 'fix_projection_db_issues',
+      -module     => 'Bio::EnsEMBL::Hive::RunnableDB::SqlCmd',
+      -parameters => {
+        db_conn   => $self->o('projection_db'),
+        sql => [
+          'UPDATE gene JOIN transcript USING(gene_id) SET transcript.biotype=gene.biotype',
+          'UPDATE protein_align_feature JOIN transcript_supporting_feature ON feature_id = protein_align_feature_id'.
+          'JOIN transcript USING(transcript_id) SET hit_name = stable_id',
+          'UPDATE protein_align_feature JOIN supporting_feature ON feature_id = protein_align_feature_id'.
+          'JOIN exon_transcript USING(exon_id) JOIN transcript USING(transcript_id) SET hit_name = stable_id',
+        ],
+      },
+      -rc_name    => 'default',
+      -flow_into  => {
+        1 => ['projection_sanity_checks'],
+      },
+    },
+
+    {
+      -logic_name => 'projection_sanity_checks',
+      -module     => 'Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveAnalysisSanityCheck',
+      -parameters => {
+        target_db => $self->o('projection_db'),
+        sanity_check_type => 'gene_db_checks',
+        min_allowed_feature_counts => get_analysis_settings('Bio::EnsEMBL::Analysis::Hive::Config::SanityChecksStatic',
+          'gene_db_checks')->{$self->o('uniprot_set')}->{'projection_coding'},
+      },
+      -rc_name    => '4GB',
+      -flow_into => {
+        1 => ['select_projected_genes'],
+      },
+    },
+
+    {
+      -logic_name => 'select_projected_genes',
+      -module     => 'Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveSelectProjectedGenes',
+      -parameters => {
+        wga_db   => $self->o('projection_db'),
+        cesar_db   => $self->o('projection_db'),
+        output_db => $self->o('selected_projection_db'),
+        dna_db => $self->o('dna_db'),
+      },
+      -rc_name    => '4GB',
+    },
+  ];
+}
+
+sub hive_capacity_classes {
+  my $self = shift;
+  return {
+    'hc_high' => 1000,
+  };
+}
+
```

Comment:
Interesting switch... :P Just to add an extra comment

Reply:
What can I say, it fails without it! But yeah, should be moved to resource classes and updated in the analysis it's used in.